test-basic/optic-vector.js

```diff
@@ -34,129 +34,191 @@ describe('tests for new vector functions.', function() {
     });
 
     it('vec.add', function(done) {
-        const vec1 = p.vec.vector([0.000000000001]);
-        const vec2 = p.vec.vector([0.000000000001, 0.000000000002]);
-        testPlan([""],p.vec.add(p.vec.subvector(vec1,0),p.vec.subvector(vec2,1)))
+        const vec1 = op.vec.vector([1]);
+        const vec2 = op.vec.vector([2]);
+        testPlan([""],op.vec.add(vec1, vec2))
             .then(function(response) {
-               assert(response.rows[0].t.value[0][0] =='1e-12')
-               assert(response.rows[0].t.value[1][0]=='2e-12')
+               // add([1], [2]) = [3] (element-wise addition)
+               assert(response.rows[0].t.value[0] == 3, 'vec.add did not return expected value: [1] + [2] should be [3]');
```

Comment:
The assertion message should remove the need for the comment above - i.e. they're both explaining the same thing. Just use an assertion message.